doc/user/content/release-notes.md

```diff
@@ -57,45 +57,133 @@ Wrap your release notes at the 80 character mark.
 - Prevent a bug that would allow `CREATE OR REPLACE` statements to create dependencies
   on objects that were about to be dropped {{% gh 5272 %}}.
 
-{{% version-header v0.6.1 %}}
+- Remove deprecated `MZ_THREADS` alias for `MZ_WORKERS`.
 
-- Add the advanced [`--timely-progress-mode` and `--differential-idle-merge-effort` command-line arguments](/cli/#dataflow-tuning)
-  to tune dataflow performance.
+{{% version-header v0.6.1 %}}
 
-- Add `ALL` to [`FETCH`](/sql/fetch).
+- Validate `WITH` clauses in [`CREATE SOURCE`](/sql/create-source) and [`CREATE
+  SINK`](/sql/create-sink) statements. Previously Materialize would ignore any
+  invalid options in these statement's `WITH` clauses.
 
-- Change [`FETCH`](/sql/fetch) with no `TIMEOUT` to wait for some rows
-  to be available.
+  Upgrading to v0.6.1 may therefore fail if any of the sources or sinks within
+  have invalid `WITH` options. If this occurs, drop these invalid sources or
+  sinks using v0.6.0 and recreate them with valid `WITH` options.
 
   **Backwards-incompatible change.**
```

Comment:
nbd, but seeing the 4 `backwards incompatible changes` right after another looks a little weird to my eyes.  (It seems to look like a title for the subsequent bullet, rather than referring to the previous text.  Wdyt of moving it to the beginning of each bullet, ie `Backwards-incompatible change: Validate WITH clause...` ?

Reply:
I'll push back gently and say this is how all of these are formatted and no one has complained about being confused by our backward-incompatible changes thus far